rsmtool/preprocessor.py

```diff
@@ -2614,6 +2609,107 @@ def process_data_rsmpredict(self, config_obj, data_container_obj):
 
         return config_obj, DataContainer(datasets)
 
+    def process_data_rsmexplain(self, config_obj, data_container_obj):
+        """
+        Process data for rsmexplain experiments.
+
+        This function takes a configuration object and a container object
+        as input and returns the same types of objects as output after
+        the loading, normalizing, and preprocessing.
+
+        Parameters
+        ----------
+        config_obj : configuration_parser.Configuration
+            A configuration object.
+        data_container_obj : container.DataContainer
+            A data container object.
+
+        Returns
+        -------
+        config_obj : configuration_parser.Configuration
+            A new configuration object.
+        data_congtainer : container.DataContainer
+            A new data container object.
+
+        Raises
+        ------
+        KeyError
+            If columns specified in the configuration do not exist in the data.
+        ValueError
+            If data contains duplicate response IDs.
+        """
+        df_background_features = data_container_obj.background_features
+        df_explainable_features = data_container_obj.explainable_features
+        df_feature_info = data_container_obj.feature_info
+
+        # get the column name that holds the IDs
+        id_column = config_obj["id_column"]
+
+        # should features be standardized?
```

Comment:
Change this?

Reply:
This comment just indicates what `standardize_features` variable represents. So what's to change?